pom.xml

```diff
@@ -101,6 +101,7 @@
             <id>pipeline-unit</id>
             <modules>
                 <module>scripted-pipeline</module>
+                <module>global-shared-library</module>
```

Comment:
Do not forget to get these moved into the main section and the `profile` deleted once hosting is complete!